### ISSUE TITLE
chore: CVEs in used go libraries

### ISSUE THREAD

**Issue by AlexSkrypnyk** [ahoy-cli/ahoy#105]
`golang/stdlib` is used by Ahoy

On Debian:
```
ENV AHOY_VERSION=2.0.2
RUN curl -L -o "/usr/local/bin/ahoy" "https://github.com/ahoy-cli/ahoy/releases/download/${AHOY_VERSION}/ahoy-bin-$(uname -s)-amd64" \
  && chmod +x /usr/local/bin/ahoy \
  && ahoy --version
```

Docker scout found these CVEs:

<img width="598" alt="Cursor_and_Image_Layer_Details_-_drevops_ci-builder_preview_1_6_0___Docker_Hub" src="https://user-images.githubusercontent.com/378794/226615018-6a3d10d5-4058-4357-91fa-702e20874982.png">

Can we please rebuild binaries using the latest go libraries. Maybe it's worth setting up renovate and CI to rebuild these automatically.

**Comment by ocean** [comment]
@AlexSkrypnyk A rebuilt release is out, [v2.1.1](https://github.com/ahoy-cli/ahoy/releases/tag/v2.1.1). I've put in the MR to Homebrew, hopefully that'll be approved overnight.

The reason for the version jump is that I found an old tag on the repo for `2.1.0`, which was from the quick cross-platform build that was done in early 2021 when I first badgered Frank about releasing for `arm64`. There was no matching release, but Homebrew packagers had picked up that older release as a more recent one because of the semver inconsistency.

So now it's `2.1.1` which is following `2.0.2`, which is a bit odd but still seems to fit.
